### PR TITLE
[11] Create iOS UI for job info view

### LINKED ISSUE #11: [iOS] [UI] As a user, I can see the job information in the job details page
## Why

As we're creating a demo to show job detail, the user should be able to read the details of a job, including price, location, time, etc before deciding to accept the jobs or not.

## Acceptance Criteria

On the job detail screen, we need to implement a scrollable area which includes:
- the price section
  - Show `Price by area` in gray
  - Show price info `80-95 Baht` in green
- the date-time section (no expandable)
  - Show section title in black
  - Show the date in black
  - Show the time in gray
- the plot info section (no expandable)
  - Show the section title in black
  - Show the distance and area info in gray for title and green for value
  - Show the address and crop type info in gray for the title and black for the value
- the message section (no expandable)
  - Show the title and value in black and gray

## Design

image

## Resources

- black: `#171923`
- gray: `#A0AEC0`
- green: `#2BC65D`

### PR DESCRIPTION
**Pull request by edgarss**
https://github.com/nimblehq/android-kmm-showcases/issues/11

## What happened 👀

Created UI for job details info view
 
## Proof Of Work 📹

Simulator Screen Shot - iPhone 13 Pro - 2022-03-24 at 21 11 54

### CODE REVIEW

File: KMM_Showcase/KMM_Showcase/Buttons.swift [hunk 1]
```diff
@@ -16,17 +16,20 @@ struct BackButton: View {
...
                     .foregroundColor(.blackColor)
                     .fontWeight(.bold)
+                    .font(.title3)
+                Spacer()
                 
             }
+//            .padding()
             .frame(maxWidth: .infinity)
-            .background(Color.white)
+//            .background(Color.white)
```
Comment (lydiasama) [inline KMM_Showcase/KMM_Showcase/Buttons.swift]: Can we get rid of the comments for the clean code? 🤔  
Or do you intend to reuse it in the future?

**Review by lydiasama — APPROVED** [review]
the rest LGTM. 👍🏻  Just an optional opinion.

File: KMM_Showcase/KMM_Showcase/Buttons.swift (hunk 1 above)
Comment (edgarss) [reply KMM_Showcase/KMM_Showcase/Buttons.swift]: forgot to clean it up :sweat_smile:

File: KMM_Showcase/KMM_Showcase/Views/JobInfoView.swift [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+struct JobInfoView: View {
+    var price = "80-95 Baht"
+    var date = "25 Mar 2022"
+    var time = "6:00-12:00"
+    var distance = "33 km"
+    var area = "70 m2"
+    var address = "123 NimbleHq"
+    var cropType = "Corn"
+    var message = "Message to pilot"
```
Comment (luongvo) [inline KMM_Showcase/KMM_Showcase/Views/JobInfoView.swift:19]: 👍

**Review by luongvo — APPROVED** [review]
All lgtm

**Pull request merged**